cuda_core/tests/conftest.py

```diff
@@ -1,18 +1,24 @@
-# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-FileCopyrightText: Copyright (c) 2024-2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
+import multiprocessing
+import pathlib
+import sys
+
 import helpers
+import pytest
 
 try:
     from cuda.bindings import driver
 except ImportError:
     from cuda import cuda as driver
-import multiprocessing
-
-import pytest
 from cuda.core.experimental import Device, _device
 from cuda.core.experimental._utils.cuda_utils import handle_return
 
+# Import shared platform helpers for tests across repos
+sys.path.insert(0, str(pathlib.Path(__file__).resolve().parents[2]))
+from cuda_python_test_helpers import IS_WSL  # noqa: F401 (imported for test modules)
```

Comment:
Is this needed in this file? Can't tests import this as needed?

Reply:
+1

I thought it's exposed based on the fact that everything listed in `conftest.py` is loaded in the scope of each test module, but it seems in one of the files we also explicitly import `IS_WSL`, then this is not needed.